workflows4s-quartz/src/test/scala/workflows4s/runtime/wakeup/quartz/QuartzKnockerUpperTest.scala

```diff
@@ -23,11 +24,27 @@ class QuartzKnockerUpperTest extends AnyFreeSpec with Matchers with BeforeAndAft
     "should schedule a wakeup at a specified time" in withQuartzKnockerUpper { knockerUpper =>
       var wokenUpAt: Instant = null
       val testId             = "test-id"
-      knockerUpper.initialize(id => IO { if (id == testId) wokenUpAt = Instant.now() }).unsafeRunSync()
-      val wakeupAt           = Instant.now().plusMillis(100)
+      knockerUpper
+        .initialize(id =>
+          IO {
+            if (id == testId) {
+              wokenUpAt = Instant.now()
+              println(s"Woken up at $wokenUpAt") // Add debug logging
+            }
+          },
+        )
+        .unsafeRunSync()
+      val wakeupAt           = Instant.now().plusMillis(500) // Increased delay to 500ms
+      println(s"Scheduling wakeup at $wakeupAt") // Add debug logging
       knockerUpper.updateWakeup(testId, Some(wakeupAt)).unsafeRunSync()
+      // Patience config is already imported at the top of the file
```

Comment:
I think that comment is redundant, if you mean the symbol (`import org.scalatest.concurrent.Eventually.PatienceConfig`)